charts/radar-kafka/values.yaml

```diff
@@ -69,6 +69,19 @@ kafka:
       cpu: 100m
       memory: 1Gi
 
+  # -- Config for the Cruise Control rebalancer (https://github.com/linkedin/cruise-control)
+  cruiseControl:
+    # -- Deploy the Cruise Control rebalancer operator.
+    enabled: false
+    # -- Add a KafkaRebalance template CRD. Note that this does not automatically rebalance the cluster.
+    # For rebalancing, you need to create a KafkaRebalance resource using the template.
+    addRebalanceTemplate: true
+    # -- Goals for the Cruise Control rebalancer.
+    # If left empty, the default goals are used.
+    goals: []
+    # -- Skip hard goal check by Cruise Control rebalancer.
+    skipHardGoalCheck: true
```

Comment:
would there be resource (cpu/mem) req/limit constraints that can be applied to this?